src/app/_components/GovernanceCalendarCards.tsx

```diff
@@ -0,0 +1,108 @@
+'use client'
+
+import { useState } from 'react'
+
+import { Clock, CalendarPlus } from '@phosphor-icons/react/dist/ssr'
+import useSWR from 'swr'
+import { z } from 'zod'
+
+import { Badge } from '@/components/Badge'
+import { CardLayout } from '@/components/CardLayout'
+import { Heading } from '@/components/Heading'
+import { TextLink } from '@/components/TextLink'
+
+import { extractTimeFromISO } from '@/utils/convertISOTime'
+
+import { GOOGLE_CALENDAR_API_URL } from '@/constants/apiUrls'
+
+type CalendarProps = {
+  startDate: string
+}
+
+function Calendar({ startDate }: CalendarProps) {
+  const { day, month } = extractTimeFromISO(startDate)
+
+  return (
+    <div className="grid min-h-40 min-w-36 grid-rows-[_40px,auto] rounded-md border border-blue-400 bg-blue-500">
+      <span className="flex items-center justify-center rounded-t text-base font-normal uppercase">
+        {month}
+      </span>
+      <span className="flex flex-1 items-center justify-center rounded-b bg-blue-100 text-5xl font-bold text-blue-400">
+        {day}
+      </span>
+    </div>
+  )
+}
```

Comment:
Think this has to be moved down, to right above `export function GovernanceCalendarCards()`.